src/server/page.html.jsx

```diff
@@ -1,35 +1,41 @@
-import { em, px } from 'csx'
 import { createMemoryHistory } from 'history'
 import React from 'react'
 import { renderToStaticMarkup } from 'react-dom/server'
 import { createTypeStyle, getStyles } from 'typestyle'
 import { AppShell } from './js/AppShell'
+import { colors } from './js/styles/common'
 
 const globalStyles = createTypeStyle() // Instantiate a different typestyle sheet since global styles won't be regenerated from the client
 
-globalStyles.cssRule('*', { lineHeight: 1.4 })
+globalStyles.cssRule('*', {
+  lineHeight: 1.4
+})
 
 // This sets the value of 1rem
-globalStyles.cssRule('html', { fontSize: px(10) })
+globalStyles.cssRule('html, body', {
+  fontSize: '16px',
```

Comment:
mixed use of font size units could get difficult to maintain maybe better to use relative units

Reply:
Leaving this here but have removed the 12pt definition below.